assets/Gruntfile.js

```diff
@@ -81,13 +81,45 @@ module.exports = function (grunt) {
           open: true,
           middleware: function (connect) {
             return [
-              modRewrite(['!^/(config.js|favicon.ico|(bower_components|scripts|images|styles|views)(/.*)?)$ /index.html [L]']),
+              modRewrite(['!^/(config.js|favicon.ico|(bower_components|scripts|images|styles|views|console|console/java)(/.*)?)$ /index.html [L]']),
+              connect().use('/config.js', function(req, res, next) {
+                var KUBERNETES_HOST = process.env.KUBERNETES_HOST || 'localhost';
+                var KUBERNETES_PORT = process.env.KUBERNETES_PORT || '8443';
+                var KUBERNETES_PROXY_VERSION = process.env.KUBERNETES_PROXY_VERSION || 'v1beta3';
+
+                var answer = 'window.OPENSHIFT_CONFIG = {' +
```

Comment:
this will be a pain to keep up to date with changes in config.js... not sure I like this